### PR TITLE
RAIN-1603: Merging JUN_SEP release changes into master from develop b…

### PR DESCRIPTION
**Pull request by rohit-eGov**
…ranch for sewerage service

### CODE REVIEW

File: sw-services/src/main/java/org/egov/swservice/service/UserService.java [hunk 1]
```diff
@@ -0,0 +1,342 @@
...
+
+					holderInfo.setId(userDetailResponse.getUser().get(0).getId());
+					holderInfo.setUuid(userDetailResponse.getUser().get(0).getUuid());
+					addUserDefaultFields(request.getSewerageConnection().getTenantId(), role, holderInfo);
+
+					StringBuilder uri = new StringBuilder(configuration.getUserHost())
+							.append(configuration.getUserContextPath()).append(configuration.getUserUpdateEndPoint());
+					userDetailResponse = userCall(new ConnectionUserRequest(request.getRequestInfo(), holderInfo), uri);
+					if (userDetailResponse.getUser().get(0).getUuid() == null) {
+						throw new CustomException("INVALID USER RESPONSE", "The user updated has uuid as null");
```
Comment (tarunlalwani) [inline sw-services/src/main/java/org/egov/swservice/service/UserService.java:80]: exception code should always have `_` and no spaces

File: sw-services/src/main/java/org/egov/swservice/web/models/Connection.java [hunk 2]
```diff
@@ -510,14 +579,18 @@ public boolean equals(java.lang.Object o) {
 				&& Objects.equals(this.connectionCategory, connection.connectionCategory)
 				&& Objects.equals(this.connectionType, connection.connectionType)
 				&& Objects.equals(this.additionalDetails, connection.additionalDetails)
-				&& Objects.equals(this.auditDetails, connection.auditDetails);
+				&& Objects.equals(this.auditDetails, connection.auditDetails)
```
Comment (tarunlalwani) [inline sw-services/src/main/java/org/egov/swservice/web/models/Connection.java:582]: Why is auditDetails compared for equality?

File: sw-services/src/main/resources/application.properties [hunk 3]
```diff
@@ -119,8 +122,11 @@ egov.receipt.businessservice=SW.ONE_TIME_FEE
...
-sw.application.pay.link=citizen/otpLogin?mobileNo=$mobileNo&redirectTo=egov-common/pay?consumerCode=$consumerCode&tenantId=$tenantId&businessService=SW.ONE_TIME_FEE
+sw.connectiondetails.link=citizen/otpLogin?mobileNo=$mobileNo&redirectTo=wns/connection-details?connectionNumber=$connectionNumber&tenantId=$tenantId&service=SEWERAGE
+sw.application.pay.link=citizen/withoutAuth/egov-common/pay?consumerCode=$consumerCode&tenantId=$tenantId&businessService=SW.ONE_TIME_FEE
+egov.msg.download.receipt.link=citizen/withoutAuth/egov-common/download-receipt?status=success&consumerCode=$consumerCode&tenantId=$tenantId&receiptNumber=$receiptNumber&businessService=$businessService&smsLink=true&mobileNo=$mobile
+egov.usr.events.download.receipt.link=citizen/otpLogin?mobileNo=$mobile&redirectTo=egov-common/download-receipt?status=success&consumerCode=$consumerCode&tenantId=$tenantId&receiptNumber=$receiptNumber&businessService=$businessService&smsLink=true&mobileNo=$mobile
+egov.usr.events.pay.link==citizen/otpLogin?mobileNo=$mobileNo&redirectTo=egov-common/pay?consumerCode=$consumerCode&tenantId=$tenantId&businessService=SW.ONE_TIME_FEE
```
Comment (tarunlalwani) [inline sw-services/src/main/resources/application.properties:129]: There is extra `==`?

**Review by tarunlalwani — CHANGES_REQUESTED** [review]

File: sw-services/src/main/java/org/egov/swservice/repository/rowmapper/SewerageRowMapper.java [hunk 4]
```diff
@@ -1,113 +1,145 @@
...
+                addtionalDetails.put(SWConstants.DETAILS_PROVIDED_BY, rs.getString("detailsprovidedby"));
+                addtionalDetails.put(SWConstants.ESTIMATION_FILESTORE_ID, rs.getString("estimationfileStoreId"));
+                addtionalDetails.put(SWConstants.SANCTION_LETTER_FILESTORE_ID, rs.getString("sanctionfileStoreId"));
+                addtionalDetails.put(SWConstants.ESTIMATION_DATE_CONST, rs.getBigDecimal("estimationLetterDate"));
+                addtionalDetails.put(SWConstants.LOCALITY, rs.getString("locality"));
+                sewarageConnection.setAdditionalDetails(addtionalDetails);
+                sewarageConnection.processInstance(ProcessInstance.builder().action((rs.getString("action"))).build());
+                sewarageConnection.setApplicationType(rs.getString("applicationType"));
+                sewarageConnection.setDateEffectiveFrom(rs.getLong("dateEffectiveFrom"));
+                sewarageConnection.setPropertyId(rs.getString("property_id"));
```
[the remark this answers is not in the document]
Comment (rohit-eGov) [reply sw-services/src/main/java/org/egov/swservice/repository/rowmapper/SewerageRowMapper.java:65]: this is a old field . But we will change it in further development

**Review by tarunlalwani — APPROVED** [review]

**Pull request merged**